config/locales/javascript/javascript.en.yml

```diff
@@ -239,9 +239,9 @@ en:
       contacts: "Contacts"
       posts: "Posts"
 
-    conversation:
-      new:
-        no_contacts: "You need to add some contacts before you can start a conversation."
```

Comment:
I wrote [here](https://github.com/diaspora/diaspora/pull/7108#discussion_r80387041) that you could use `t("javascripts.conversation.new.no_contacts")`.